src/plugins/ui_feature/cell_popovers.ts

```diff
@@ -84,6 +88,10 @@ export class CellPopoverPlugin extends UIPlugin {
         };
   }
 
+  hasOpenedPopover() {
+    return this.persistentPopover != null;
```

Comment:
what if `this.persistentPopover` is `undefined` ?

Reply:
> what if `this.persistentPopover` is `undefined` ?

@LucasLefevre `undefined == null` is true so if `this.persistentPopover` is undefined this will return `false`, which should make sense? (Unless when `this.persistentPopover` is `undefined` it means having opened popover).

Reply:
Oh yeah didn't pay attention you used `!=` instead of `!==`
Using `!=` and `==` are error prone and lead to unexpected results.

I think we should never use those operators and be explicit in the predicate expression